converter.go

```diff
@@ -0,0 +1,15 @@
+package katsubushi
+
+import "time"
+
+func ToTime(id uint64) time.Time {
+	ts := id>>(WorkerIDBits+SequenceBits) + TimestampSince
+	sec := ts / 1000
+	msec := ts % 1000
+	return time.Unix(int64(sec), int64(msec)*int64(time.Millisecond))
```

Comment:
:memo:

- time.Unix() receives sec and nano sec.  ([doc](https://golang.org/pkg/time/#Unix))
- 1 unit of time.Duration = nanosecond. ([src](https://golang.org/pkg/time/#Duration))